.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
Feel free to ditch this, but I felt it would be useful for anyone out there who may be using IDEs outside of sublime or similar.